earth_enterprise/SConstruct

```diff
@@ -24,6 +26,17 @@ from khEnvironment import khEnvironment
 
 EnsureSConsVersion(0, 96, 92)
 
+# Set 'package_command_arguments' based on a parameter
+# that is to be passed on to Gradle
+package_command_arguments = "-Pdebug"
+
+for parameter in [ 'release', 'optimize', 'profile' ] :
+  if int( ARGUMENTS.get( parameter, '0' ) ) > 0 :
```

Comment:
This doesn't need to be `> 0` to handle cases when a parameter is `0`.  This implementation, actually, makes it inconsistent with how it's parsed in `src/SConstruct`.  Negative values are interpreted as `True` there and `False` here.